### DISCUSSION TITLE
Should Zed employees be a "Member" of the Zed organization?

### DISCUSSION THREAD

**Discussion by baldwindavid** [zed-industries/zed#8847] [Feature Requests]
I'm curious if y'all have considered adding Zed employees (or those listed at https://zed.dev/team) as members of the "zed-industries" organization. I've been on here for a bit so I typically know when I'm reading a comment from an employee or not, but sometimes I have to double-check. I would imagine it's much less clear for those new to the project.

It can be handy to know how authoritative, knowledgeable, etc a given comment is if the "Member" badge is displayed.

![Screenshot 2024-03-04 at 1 25 26 PM](https://github.com/zed-industries/zed/assets/4578/05b6d62e-2f49-48c9-811e-97aeae21e624)

While these employees are at least marked as "Contributor", so is anyone else that has gotten a commit into the zed repo...

![Screenshot 2024-03-04 at 1 20 28 PM](https://github.com/zed-industries/zed/assets/4578/8f1b6002-6ac9-422b-9ab6-e1faebec189a)

It appears that only two devs are actual members of the org in Github and one is not still on the Team page of zed.dev.

No big deal either way, but might eliminate a bit of confusion. Thanks.

**Comment by maxdeviant** [comment]
I think this comes down to individual users' settings for their organization membership visibility.

I just noticed that mine was set to "Private" still, so I modified it:

<img width="393" alt="Screenshot 2024-04-03 at 11 28 58 AM" src="https://github.com/zed-industries/zed/assets/1486634/8590bbd6-435b-4f43-8a6c-308e1e4b7feb">

Interestingly, I see Zed Staff as "Member" because since I'm a member of the team I can see private team memberships (although if I go into an incognito window then I only see "Member" for this with public memberships).
